src/authorize.ts

```diff
@@ -45,6 +45,9 @@ export interface IEveryAuthOptions {
    * constant for experimentation and testing purposes.
    *
    * The tenant represents the larger customer, which may include multiple authorized users.
+   *
+   * If mapToTenantId is not supplied, then the tenantId on an identity will be set to the same value as the
+   * userId.
```

Comment:
rephrase to: If you don't supply mapToTenantId, the tenantId on an identity will be the same value as the userId.